jira-frontend/src/app/app-routing.module.ts

```diff
@@ -19,6 +20,9 @@ const routes: Routes = [
       {
        path:'settings/:projectId', component: ProjectSettingsComponent
       },
+      {
+        path:'issues/create', component: IssueCreateComponent
+      },
```

Comment:
Route Heirarcy can be created istead of flat routes.
Example:
const routes: Routes = [
  { path: "", redirectTo: "home", pathMatch: "full" },
  { path: "find", redirectTo: "search" },
  { path: "home", component: HomeComponent },
  { path: "search", component: SearchComponent },
  {
    path: "artist/:artistId",
    component: ArtistComponent,
    children: [
      { path: "", redirectTo: "tracks", pathMatch: "full" },
      { path: "tracks", component: ArtistTrackListComponent },
      { path: "albums", component: ArtistAlbumListComponent }
    ]
  },
  { path: "**", component: HomeComponent }
];